From adf1849bf993a337f1484596fbe41b74064b749f Mon Sep 17 00:00:00 2001
From: Fedora dracut team <dracut-maint@redhat.com>
Date: Tue, 3 Mar 2015 09:26:02 +0100
Subject: [PATCH] lvm: also install /etc/lvm/lvm_hostname.conf

enabling hosttags in /etc/lvm/lvm.conf causes new ramdisks to fail to
boot, if no lvm_hostname.conf is specified.

https://bugzilla.redhat.com/show_bug.cgi?id=1130565
---
 modules.d/90lvm/install | 1 +
 1 file changed, 1 insertion(+)

diff --git a/modules.d/90lvm/install b/modules.d/90lvm/install
index 5c7e758..6b507be 100755
--- a/modules.d/90lvm/install
+++ b/modules.d/90lvm/install
@@ -6,6 +6,7 @@ inst_rules "$moddir/64-lvm.rules" 64-lvm.rules
 if [[ $hostonly ]] || [[ $lvmconf = "yes" ]]; then 
     if [ -f /etc/lvm/lvm.conf ]; then
 	inst /etc/lvm/lvm.conf
+	[ -f /etc/lvm/lvm_hostname.conf ] && inst /etc/lvm/lvm_hostname.conf
 	# FIXME: near-term hack to establish read-only locking;
 	# use command-line lvm.conf editor once it is available
 	sed -i -e 's/\(^[[:space:]]*\)locking_type[[:space:]]*=[[:space:]]*[[:digit:]]/\1locking_type = 4/' ${initdir}/etc/lvm/lvm.conf
